modules/hdf5_handler/h5commoncfdap.cc

```diff
@@ -724,41 +723,27 @@ void gen_dap_onevar_dmr(libdap::D4Group* d4_grp, const HDF5CF::Var* var, const h
         BaseType *bt = nullptr;
 
         switch (var->getType()) {
-            // TODO Remove extra ';' jhrg 3/9/22
 #define HANDLE_CASE(tid,type)                                  \
             case tid:                                           \
                 bt = new (type)(var->getNewName(),var->getFullPath()); \
             break;
```

Comment:
Another way to handle this would be to use an inline template function. This way, using a macro, is fine, but I thought I'd mention the alternative for completeness sake.

Reply:
I will talk with you about this when I have another round of clean-up.